examples/nlp/glue_benchmark/glue_benchmark_with_bert.py

```diff
@@ -31,6 +31,8 @@
 --data_dir /path_to_data_dir/MRPC \
 --task_name mrpc \
 --work_dir /path_to_output_folder \
+--pretrained_model_name roberta-base \
```

Comment:
could you make bert as default, and if you use roberta/albert add a note in the comment that there are also other alternatives?

Reply:
done